src/CMakeLists.txt

```diff
@@ -159,14 +159,21 @@ elseif(CMAKE_CXX_COMPILER_ID STREQUAL Clang)
     -Wno-gnu-designator
     -Wno-missing-braces
     -Wno-parentheses
-    -Wno-deprecated-register)
+    -Wno-deprecated-register
+    -Wno-reserved-user-defined-literal)
```

Comment:
clang specific. I would like to avoid this but it seems like all other repositories like to concatenate string literal definitions without a whitespace. So I just wanted to conform instead of adding another 100 files with changes (I have them stashed).

This is about the code like 
```
	fprintf(f, "    put_bulk=%"PRIu64"\n", sum.put_bulk);
	fprintf(f, "    put_objs=%"PRIu64"\n", sum.put_objs);
	fprintf(f, "    get_success_bulk=%"PRIu64"\n", sum.get_success_bulk);
	fprintf(f, "    get_success_objs=%"PRIu64"\n", sum.get_success_objs);
	fprintf(f, "    get_fail_bulk=%"PRIu64"\n", sum.get_fail_bulk);
	fprintf(f, "    get_fail_objs=%"PRIu64"\n", sum.get_fail_objs);
	if (info.contig_block_size > 0) {
```

note that there is no whitespace between `"` and `PRIu64`. This is being barked at by the latest clang compiler, but there are plenty of such instances.

Reply:
Not sure I understand why this was not noticed till now. Clang is used to compile Ceph on
many op-sys, inc. Fedora 38.

Reply:
maybe some default warning settings? This problem is described here: https://en.cppreference.com/w/cpp/language/user_literal#Notes

> Since the introduction of user-defined literals, the code that uses [format macro constants for fixed-width integer types](https://en.cppreference.com/w/c/types/integer) with no space after the preceding string literal became invalid: `std::printf("%"PRId64"\n",INT64_MIN);` has to be replaced by `std::printf("%" PRId64"\n",INT64_MIN]);`.